lib/perl/Genome/InstrumentData/Command/Import/Basic_bam_multi_groups.t

```diff
@@ -78,7 +81,8 @@ for my $rg_id ( sort keys %expected_read_groups ) {
     ok(-s $bam_path, 'bam path exists');
     is($bam_path, $instrument_data->data_directory.'/all_sequences.bam', 'bam path correctly named');
     is(eval{$instrument_data->attributes(attribute_label => 'bam_path')->attribute_value}, $bam_path, 'set attributes bam path');
-    my $bam_basename = join('.', 'all_sequences', 'basic-multi-rg', $expected_read_groups{$rg_id}->[0], $paired_key, 'bam');
+    my $bam_basename = join('.', 'all_sequences', 'basic-multi-rg', $expected_read_groups{$rg_id}->[0], $type, 'bam');
+    print "gvimdiff $bam_path $test_dir/$bam_basename\n";
```

Comment:
I'd remove this `print` or at least the `gvimdiff` part!  (Otherwise, same as my other `print`-related comment.)